bindings/python/docs/source/developer/installation.rst

```diff
@@ -12,7 +12,7 @@ Command Line Tools. Additionally, you need CMake and pkg-config::
   $ brew install cmake
   $ brew install pkg-config
 
-On Linux, you require gcc 4.8, CMake and pkg-config.
+On Linux, you require gcc 12, CMake and pkg-config.
```

Comment:
Minor nitpick, but we should probably change the wording to "installation requires ..."

Reply:
Fixed.